### ISSUE TITLE
Upgrade to Grunt 1.0 and NodeJS 4.0.0 causes failure

### ISSUE THREAD

**Issue by generalconsensus** [linusnorton/grunt-parallel-behat#17]
We are encountering build failures due to the upgrade to Grunt 1.0 and Node 4.0. 

Can we move the upgrade to a separate non-mainline branch?

```

npm WARN peerDependencies The peer dependency grunt@>=1.0.0 included from grunt-parallel-behat will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency 
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.
npm ERR! Linux 2.6.32-573.22.1.el6.x86_64
npm ERR! argv "/opt/node-v0.10.28-linux-x64/bin/node" "/usr/local/bin/npm" "install"
npm ERR! node v4.4.1
npm ERR! npm  v2.14.20
npm ERR! code EPEERINVALID

npm ERR! peerinvalid The package grunt@0.4.5 does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer grunt-sass-globbing@1.4.0 wants grunt@^0.4.5
npm ERR! peerinvalid Peer grunt-contrib-coffee@0.13.0 wants grunt@>=0.4.0
npm ERR! peerinvalid Peer grunt-contrib-concat@0.5.1 wants grunt@>=0.4.0
npm ERR! peerinvalid Peer grunt-contrib-copy@0.8.2 wants grunt@>=0.4.0
npm ERR! peerinvalid Peer grunt-parallel-behat@0.4.3 wants grunt@>=1.0.0
npm ERR! peerinvalid Peer grunt-postcss@0.7.1 wants grunt@^0.4.5
npm ERR! peerinvalid Peer grunt-contrib-watch@0.6.1 wants grunt@~0.4.0
npm ERR! peerinvalid Peer grunt-contrib-jshint@0.11.3 wants grunt@>=0.4.0
npm ERR! peerinvalid Peer grunt-contrib-nodeunit@0.4.1 wants grunt@~0.4.0
npm ERR! peerinvalid Peer grunt-contrib-cssmin@0.12.3 wants grunt@>=0.4.0
npm ERR! peerinvalid Peer grunt-contrib-uglify@0.9.2 wants grunt@>=0.4.0
npm ERR! peerinvalid Peer grunt-contrib-compass@1.0.4 wants grunt@>=0.4.0

**Comment by linusnorton** [comment]
@generalconsensus I've pushed grunt back to 0.4.x on master. Can you check that it works for you?

If it works I will push it as 0.4.4 but I will am going to release version 1.0.0 with grunt version 1.0.0. You can set the version you want to use in your package.json. 

I would also recommend upgrading to grunt 1.0 it's quite painless.

**Comment by linusnorton** [comment]
@generalconsensus any update?

**Comment by generalconsensus** [comment]
@linusnorton Apologies for the delay. I will follow up with you today

**Comment by generalconsensus** [comment]
Just to be clear two things will happen on this repo:
- Branch 1.0.0 will be created which pushes us to Grunt stable 1.0.0, this will be the registered as the default branch with npmjs so that a package.json set with the following will install with grunt 1.0.0 as a requirement 

```
  "devDependencies": {
    "grunt-parallel-behat": "^1.0.0"
  }
```

OR this will work as well:

```
  "devDependencies": {
    "grunt-parallel-behat": "*"
  }
```

- Branch 0.4 will be created, which allows for backward compatibility, sets the required Grunt version to ^0.4.0. Setting the package.json to the following installs grunt-parallel-behat to use Grunt version ^0.4.0:

```
  "devDependencies": {
    "grunt-parallel-behat": "^0.4.0"
  }
```

Is that correct?

**Comment by linusnorton** [comment]
That's correct

**Comment by linusnorton** [comment]
I've now published version 0.4.4 based on the version 0.4.0 of grunt and also version 1.0.0 for grunt version 1.0.0 or above. Please pin your project appropriately. Let me know if you have any problems

**Comment by generalconsensus** [comment]
Thanks @linusnorton! All is well again -- I will upgrade to 1.0... eventually ;)
